<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
  <!-- Generated for PYSIDE6 -->
  <class>SizeTest</class>
  <widget class="QDialog" name="SizeTest">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>400</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>SizeTest</string>
    </property>
    <widget class="QPushButton" name="expandingButton">
      <property name="text">
        <string>Expanding Button</string>
      </property>
      <property name="geometry">
        <rect>
          <x>50</x>
          <y>50</y>
          <width>200</width>
          <height>40</height>
        </rect>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed" />
      </property>
      <property name="minimumSize">
        <size>
          <width>100</width>
          <height>30</height>
        </size>
      </property>
      <property name="maximumSize">
        <size>
          <width>400</width>
          <height>50</height>
        </size>
      </property>
    </widget>
  </widget>
</ui>
